src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -384,136 +370,56 @@ def handlePileup(self):
         siteConfig = loadSiteLocalConfig()
         PhEDExNodeName = siteConfig.localStageOut["phedex-node"]
         self.logger.info("Running on site '%s', local PNN: '%s'", siteConfig.siteName, PhEDExNodeName)
+        jsonPileupConfig = os.path.join(self.stepSpace.location, "pileupconf.json")
 
-        pileupDict = self._getPileupConfigFromJson()
-
-        # 2011-02-03 according to the most recent version of instructions, we do
```

Comment:
If we don't drop the multiple pileup types, I think it would be useful to keep some of these docstrings around.
Same comment for a few other docstring blocks below (like the one about AAA, etc).

Reply:
Agreed. I will reincorporate the docstrings